### ISSUE TITLE
Does not capture passwords

### ISSUE THREAD

**Issue by kuza55** [caseyscarborough/keylogger#3]
This is probably a known issue, but this keylogger doesn't capture passwords, probably due to EnableSecureEventInput

This seems unavoidable with this approach, but would be nice if it was mentioned in the README

https://developer.apple.com/library/mac/technotes/tn2150/_index.html

**Comment by sushain97** [comment]
Is there any way to get around this limitation?

**Comment by caseyscarborough** [comment]
This is not possible, any secure input fields will not be captured. Closing for now unless this becomes possible.

**Comment by ghostlymedic** [comment]
Agreed this would be helpful in the README.

**Comment by caseyscarborough** [comment]
@ghostlymedic Good point, I will update the README with this information... Also, I'll reopen this issue to give visibility, it seems to get duplicated often.

**Comment by jondb** [comment]
@caseyscarborough the technote linked above suggests that you can call `IsSecureEventInputEnabled` and `DisableSecureEventInput`. Do those calls return global system state or modify global system state?

If `EnableSecureEventInput` affects the global system state, does `DisableSecureEventInput` also affect global state? That is can you turn off secure input from a background process?

**Comment by 1mpossible-code** [comment]
i like cock
